### PR TITLE
Update fund.mdx with Sepolia details

### PR DESCRIPTION
**Pull request by jlwllmr**
This PR updates the "Fund your accounts" page in `use-mainnet` with an additional tab for Sepolia. It includes a link to the transition page where faucets are listed, and a set of instructions detailing how to bridge from L1 Sepolia using the native bridge.

### CODE REVIEW

File: docs/use-mainnet/fund.mdx [hunk 1]
```diff
@@ -142,24 +167,64 @@ If you already have tokens on another network, you can [follow the instructions
...
+</TabItem>
 
-4. **Set the payment method**. Again, if you've used Buy before on your current device, your previous choice should be loaded by default.
+<TabItem value="Linea Sepolia">
 
-5. **Select a token**. The token selection menu also allows you to switch networks; it's here where you need to locate and select Linea from the networks list.
+## Linea Sepolia faucets
 
-6. Now you'll see the tokens available to buy on Linea. Pick the one you want to buy, then input the amount on the main screen once the tokens menu is gone.
+An up-to-date list of available Linea Sepolia faucets is available [here](../build-on-linea/goerli-to-sepolia.mdx#faucets).
```
Comment (emilyJLin95) [inline docs/use-mainnet/fund.mdx:200]: Is there a reason we are redirecting to the other page rather than putting it directly on this page

Comment (jlwllmr) [reply docs/use-mainnet/fund.mdx:200]: No reason other than it'll prevent us having to update the information in two places. The faucets aren't all currently live so the information will need updating in `goerli-to-sepolia` at some point. Appreciate though that being sent round the docs isn't ideal UX.

File: docs/use-mainnet/fund.mdx [hunk 2]
```diff
@@ -24,20 +32,25 @@ There are several ways to get test tokens on Linea:
...
 - [Bridging between Goerli and Linea Goerli](./bridges-of-linea/index.mdx)
-- Sending a DM to [this XMTP chatbot](https://converse.xyz/dm/0x96561c772533d32da71d240820a2fbf6883cf96a) at TheGeneralStore.eth
+- Sending a DM to [this XMTP chatbot](https://converse.xyz/dm/0x96561c772533d32da71d240820a2fbf6883cf96a) 
+at TheGeneralStore.eth
 
 ## Use a Linea faucet
 
-If you want to drip Goerli ETH directly to Linea testnet, the following faucets are available. Note that you will need to enter your actual address — ENS names will not work.
+If you want to drip Goerli ETH directly to Linea testnet, the following faucets are available. Note
+that you will need to enter your actual address — ENS names will not work.
```
Comment (bgravenorst) [inline docs/use-mainnet/fund.mdx:41]:
```suggestion
If you want to drip Goerli ETH directly to the Linea testnet, use the following faucets. Note that you must enter your actual address — ENS names will not work.
```

File: docs/use-mainnet/fund.mdx [hunk 3]
```diff
@@ -24,20 +32,25 @@ There are several ways to get test tokens on Linea:
...
 
 1. [Infura Linea faucet](https://infura.io/faucet/linea)
 2. [Covalent Linea faucet](https://www.covalenthq.com/faucet/)
 3. [FAUCETME faucet](https://linea.faucetme.pro/)
 4. [LearnWeb3 Faucet](https://learnweb3.io/faucets/linea_goerli)
 
-If you want more ETH than the daily allotted amount or run into trouble with the above faucets, you can also [bridge ETH to Linea](/use-mainnet/bridges-of-linea/how-to-bridge-eth). Note that you'll need to [get test ETH on Goerli](#get-test-eth-on-goerli) in order to do so.
+If you want more ETH than the daily allotted amount or run into trouble with the above faucets, you
+can also [bridge ETH to Linea](/use-mainnet/bridges-of-linea/how-to-bridge-eth). Note that you'll 
+need to [get test ETH on Goerli](#get-test-eth-on-goerli) in order to do so.
```
Comment (bgravenorst) [inline docs/use-mainnet/fund.mdx:50]:
```suggestion
If you want more ETH than the daily allotted amount, or run into trouble with the above faucets, you
can also [bridge ETH to Linea](/use-mainnet/bridges-of-linea/how-to-bridge-eth). Note that you'll 
need to [get test ETH on Goerli](#get-test-eth-on-goerli) to do so.
```

File: docs/use-mainnet/fund.mdx [hunk 4]
```diff
@@ -24,20 +32,25 @@ There are several ways to get test tokens on Linea:
...
 
-If you want to drip other tokens, you can find the [multi-token Linea faucet here](https://faucet.goerli.linea.build/), which lists the different tokens you can add to your wallet on the Goerli and Linea Goerli testnet.
+If you want to drip other tokens, you can find the [multi-token Linea faucet here](https://faucet.goerli.linea.build/), 
+which lists the different tokens you can add to your wallet on the Goerli and Linea Goerli testnet.
```
Comment (bgravenorst) [inline docs/use-mainnet/fund.mdx:53]:
```suggestion
If you want to drip other tokens, use the [multi-token Linea faucet](https://faucet.goerli.linea.build/), 
which lists the different tokens you can add to your wallet on the Goerli and Linea Goerli testnet.
```

**Review by bgravenorst — COMMENTED** [review]
Minor suggestions. Lgtm

**Review by bgravenorst — APPROVED** [review]
Forgot to approve :)

**Comment by bgravenorst** [comment]
FYI, the `https://infura.io/faucet/linea` link fails due to the website verifying the security of your connection. So I added it to the config file in my PR #518.

Also suggest using relative paths to fix the other error.

**Pull request merged**